### PR TITLE
Add man_made=tower

### PR DESCRIPTION
**Pull request by pastk**
- https://github.com/organicmaps/organicmaps/issues/5521

### CODE REVIEW

**Comment by pastk** [comment]
@vng could you please check my changes to the tests?

File: search/types_skipper.cpp [hunk 1]
```diff
@@ -21,10 +21,8 @@ TypesSkipper::TypesSkipper()
   for (auto const & e : arrSkipEmptyName1)
     m_skipIfEmptyName[0].push_back(c.GetTypeByPath(e));
 
-  // Test for exact type (man_made-tower-communication is not).
   StringIL const arrSkipEmptyNameExact[] = {
     {"man_made", "chimney"},
-    {"man_made", "tower"},
```
Comment (vng) [inline search/types_skipper.cpp:27]: Just noticed here. IMO, there is no need to search for a regular unnamed "tower" by category search.

Comment (pastk) [reply search/types_skipper.cpp:27]: By this logic we can potentially skip some other unnamed categories, incl. masts, water towers..

Comment (biodranik) [reply search/types_skipper.cpp:27]: What is displayed in Paris with this PR and "Eiffel Tower" query?

**Pull request merged**